### ISSUE TITLE
build tab not loading

### ISSUE THREAD

**Issue by molliejmcdowell** [STAT545-UBC/Discussion#281]
Hi,

I'm on my second peer review for hw 8 and the build tab is not showing up when I open the makefile. This didn't happen with the first one. Is this a known problem/is there a possible solution to this that doesn't involve me actually changing the file into a makefile? I suppose it could be that the file was never actually made into a makefile.

**Comment by daattali** [comment]
You might have to set the option in RStudio to make this project used Make.  Go to *Tools > Project Options > Build Tools* and select *Makefile*.  Maybe that'll fix it

**Comment by jennybc** [comment]
What is the name of the makefile? Where is it relative to top-level directory of the Project? I think certain things have to be true auto-detection to work.

**Comment by molliejmcdowell** [comment]
It is called "Makefile" and is within the "hw9" folder. The hw9 folder is within the repository that contains the project for the course. There aren't projects within projects or anything like that.

**Comment by molliejmcdowell** [comment]
Hi, sorry to harp on this,
I turned "Makefile" into a makefile manually in rstudio, but clean and rebuild told me that the "Build directory does not contain a makefile." The Makefile is in the homework folder whereas the build directory is the parent folder, where the project is. I moved Makefile into the parent folder and everything works fine now, so I'm assuming what happened was that my peer had the makefile working and then moved all the files into the homework folder and didn't check to make sure it could still clean and rebuild.

Which leads me to a question I had earlier, and sorry if someone has already answered this - do the Makefile and supporting files need to be in the top-level project folder? This is why I made a new repository for my Makefile, but I wasn't sure if there's another solution.
